addons/crm/tests/test_crm_lead_notification.py

```diff
@@ -12,7 +12,9 @@ class NewLeadNotification(TestCrmCommon):
     @users('user_sales_manager')
     def test_lead_message_get_suggested_recipient(self):
         """ Test '_message_get_suggested_recipients' and its override in lead. """
-        lead_format, lead_multi, lead_from, lead_partner = self.env['crm.lead'].create([
+        partner_no_email = self.contact_2
+        partner_no_email.email = False
+        lead_format, lead_multi, lead_from, lead_partner, lead_partner_no_email, lead_partner_no_email_with_cc = self.env['crm.lead'].create([
```

Comment:
Maybe line could be split, like
```suggestion
        (
            lead_format, lead_multi, lead_from,
            lead_partner, lead_partner_no_email,
            lead_partner_no_email_with_cc
        ) = self.env['crm.lead'].create([
```